actioncable/test/subscription_adapter/redis_test.rb

```diff
@@ -14,3 +14,9 @@ def cable_config
     super.merge(driver: "hiredis")
   end
 end
+
+class RedisAdapterWithChannelPrefixTest < RedisAdapterTest
```

Comment:
I'm having trouble writing tests with two differently configured adapters as `ActionCable::Server::Base`'s config is defined as a cattr_accessor, so when I make a second one & give it a new config, it shares config with the first.